packages/sdk/src/create.ts

```diff
@@ -6,6 +6,15 @@ import { CreateWakuNodeOptions, WakuNode, WakuOptions } from "./waku.js";
 
 export { Libp2pComponents };
 
+export async function createApplicationNode(
```

Comment:
at this point we have way to many `create*` functions

for now let's have only `createNode` and extend with additional parameter in `options`

Reply:
@adklempner do you need support with this one? we can go ahead with this PR and follow up later as we discussed at prev PM